nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -379,6 +379,8 @@ Available as [services.patroni](options.html#opt-services.patroni.enable).
 
 - There is a new module for the `xfconf` program (the Xfce configuration storage system), which has a dbus service.
 
+- The Mastodon package got upgraded from the major version 3 to 4. See the [v4.0.0 release notes](https://github.com/mastodon/mastodon/releases/tag/v4.0.0) on migration recommendations.
+
```

Comment:
In our case, the migration is done automatically.

Reply:
Ah right, we should add something to not let users have the expectation that they *have* to run anything by themself.

Reply:
Better now?

Reply:
If `services.mastodon.automaticMigrations = true`, then migration is performed automatically.

Reply:
You could add that the user might want to backup the db.